assets/sass/components/_reviews-table.scss

```diff
@@ -1,4 +1,46 @@
 .app-reviews-table {
+  thead {
```

Comment:
loosely based on [MoJ Frontend sortable table component](https://github.com/ministryofjustice/moj-frontend/tree/bc848ffdd0813814b806f4e223430badea7186ea/src/moj/components/sortable-table) but that sorts in-browser so cannot be used for our purposes